src/app/components/SearchResults/SearchResultsContainer.jsx

```diff
@@ -52,11 +65,12 @@ const SearchResultsContainer = (props, context) => {
           aria-describedby="results-description"
         >
           {
-            !!(results && results.length !== 0) &&
-            <ResultsList
-              results={results}
-              searchKeywords={searchKeywords}
-            />
+            results && results.length !== 0 ?
```

Comment:
is this checking for the same thing as https://github.com/NYPL-discovery/discovery-front-end/blob/f3490b3d55a4f791021098bbf83f22ecca95b588/src/app/components/SearchResults/SearchResultsContainer.jsx#L77 ?

Reply:
Checked out the git blame on this. This is from before my time. Good call. I tweak the logic, but committed it to SCC-2113 without thinking much of it. Not a big deal. It was just code style not relevant to this PR.